src/translations/en.yml

```diff
@@ -0,0 +1,2 @@
+Continue: ''
+Cancel: ''
```

Comment:
Is this correct? I haven't had a chance to look into i18next enough yet, but I was hoping we could setup the generation to be better than how it is in cloud (manual) and cms (I think same as here). Benefits of this current way is the Japanese will be easier to translate (bc the english can be seen in ja.yml), so maybe this way is ok? Just `EnglishWord: ''` seems a little strange to me.

Reply:
Setting useKeysAsDefaultValue to true will cause the key name to be inserted directly into the value.